packages/vuetify/src/components/VProgressCircular/VProgressCircular.tsx

```diff
@@ -0,0 +1,142 @@
+// Styles
+import './VProgressCircular.sass'
+
+// Composables
+import { makeTagProps } from '@/composables/tag'
+import { useTheme } from '@/composables/theme'
+import { useColor } from '@/composables/color'
+
+// Utilities
+import { computed, defineComponent } from 'vue'
+import { convertToUnit, makeProps } from '@/util'
+
+export default defineComponent({
+  name: 'VProgressCircular',
+
+  props: makeProps({
+    indeterminate: Boolean,
+    rotate: {
+      type: [Number, String],
+      default: 0,
+    },
+    size: {
+      type: [Number, String],
+      default: 32,
+    },
+    color: {
+      type: String,
+      default: 'primary',
+    },
+    width: {
+      type: [Number, String],
+      default: 4,
+    },
+    value: {
+      type: [Number, String],
+      default: 0,
+    },
+    ...makeTagProps({ tag: 'div' }),
+  }),
+
+  setup (props, { slots }) {
+    const radius = 20
```

Comment:
should this be a static value?

Reply:
It's used with the loading animation, I think to determine how much of the circle is filled, and how much is gapped. It was static in the previous version, and when changing it, it isn't intuitive what happens when the number changes. With 10, the gap is smaller, with 100 you get 3 different section loading and 3 gaps.